setuptools/dist.py

```diff
@@ -529,7 +529,7 @@ def make_option_lowercase(self, opt, section):
             future versions. Please use lowercase {lowercase_opt!r} instead.
             """,
             see_docs="userguide/declarative_config.html",
-            due_date=(2024, 9, 26),
+            due_date=(2025, 3, 3),
```

Comment:
Should be discussed in a separated issue/PR. For now just postponing to get the CI error out of the way.